concent_api/core/tests/test_payments_bankster.py

```diff
@@ -256,208 +257,327 @@ def test_that_when_there_are_other_deposit_claims_finalize_payment_substract_the
-                    requestor_public_key=hex_to_bytes_convert(task_to_compute.requestor_public_key),
-                )
+    def create_client_and_related_deposit_account(self):
+        self.requestor_client = Client(
+            public_key=self.task_to_compute.requestor_id
+        )
+        self.requestor_client.full_clean()
+        self.requestor_client.save()
+
+        self.requestor_deposit_account = DepositAccount(
+            client=self.requestor_client,
+            ethereum_address=self.task_to_compute.requestor_ethereum_address,
+        )
+        self.requestor_deposit_account.full_clean()
+        self.requestor_deposit_account.save()
+
+    def create_deposit_claim(self, amount, tx_hash):
+        deposit_claim = DepositClaim(
+            payee_ethereum_address=self.task_to_compute.provider_ethereum_address,
+            payer_deposit_account=self.requestor_deposit_account,
+            amount=amount,
+            concent_use_case=ConcentUseCase.FORCED_PAYMENT,
+            tx_hash=tx_hash,
+            closure_time=parse_timestamp_to_utc_datetime(get_current_utc_timestamp()),
+        )
+        deposit_claim.full_clean()
+        deposit_claim.save()
+
+    def call_settle_overdue_acceptances_with_mocked_sci_functions(
+        self,
+        get_deposit_value_return_value=None,
+        get_list_of_payments_return_value=None,
+    ):
+        with freeze_time("2018-02-05 10:00:25"):
+            with mock.patch('core.payments.bankster.validate_list_of_transaction_timestamp') as self.validate_list_of_transaction_mock:
+                with mock.patch(
+                    'core.payments.bankster.service.get_deposit_value',
+                    return_value=(
+                        get_deposit_value_return_value if get_deposit_value_return_value is not None
+                        else self.get_deposit_value_return_value_default
+                    ),
+                ) as self.get_deposit_value_mock:
+                    with mock.patch(
+                        'core.payments.bankster.service.get_list_of_payments',
+                        return_value=(
+                            get_list_of_payments_return_value if get_list_of_payments_return_value is not None
+                            else self._get_list_of_force_transactions()
+                        ),
+                    ) as self.get_list_of_payments_mock:
+                        claim_against_requestor = settle_overdue_acceptances(
+                            requestor_ethereum_address=self.task_to_compute.requestor_ethereum_address,
+                            provider_ethereum_address=self.task_to_compute.provider_ethereum_address,
+                            acceptances=self.subtask_results_accepted_list,
+                            requestor_public_key=hex_to_bytes_convert(self.task_to_compute.requestor_public_key),
+                        )
+
+        return claim_against_requestor
+
+    def assert_mocked_function_were_called(self):
```

Comment:
please rename to `assert_mocked_sci_functions_were_called`. 

BTW: :+1: for code extraction ;)

Reply:
:+1: done